### DISCUSSION TITLE
<Your question or feedback about Sunshine>

### DISCUSSION THREAD

**Discussion by MuJian669** [LizardByte/community#574] [Sunshine]
### Select Topic Area

Question

### Body

Some networks of sunshine in the latest preview version 2024.1115.143556 cannot be directly connected via IPV6. It prompts timeout, error 0, and prompts to check tcp47984 47989.
![error0](https://github.com/user-attachments/assets/3f5f9208-622d-4795-bf89-8ed4f1705e9c)
 However, the old version 0.23.1 is completely normal. The configuration is retained and upgraded to the new version. Is there any change in the new version network? Is the port number changed?

**Comment by ReenigneArcher** [comment]
port numbers did not change

**Reply by MuJian669** [reply]
Very strange, 0.23.1 is normal, but the latest version is not normal...

[a comment by an automated account is not shown]
